iree/compiler/Dialect/Flow/Transforms/Passes.cpp

```diff
@@ -73,6 +67,19 @@ namespace IREE {
 namespace Flow {
 
 void buildFlowTransformPassPipeline(OpPassManager &passManager) {
+  // Special case peephole optimizations.
+  {
+    passManager.addNestedPass<FuncOp>(createConvertConv2D1x1ToMatmulPass());
+    if (clEnableConvToImg2Col) {
```

Comment:
Could we also drop these two flags in favor of using `iree-opt` before invoking `iree-translate`?

Reply:
There are few things need before we kill these flags. First to get `iree-opt` to run `-iree-common-input-transformation-pipeline`, second change benchmarks to invoke `iree-opt` then we can kill these flags.